### DISCUSSION TITLE
Unable to get STOMP setup running

### DISCUSSION THREAD

**Discussion by M0ulinamary** [rabbitmq/rabbitmq-server#8139] [Other]
### Describe the bug

I have done the rabbitmq setup using the readily available Docker image rabbitmq:3.11-management. Now i could see that ports  5672 and 15672 are in listening mode.  Now, i want to enable STOMP plugin  which must listen to 61613 port.

But, after enabling, i am unable to see the 61613 port listening. Also, unable to telnet to the RabbitMQ/Stomp setup from my controller.

Could you please help ?

### Reproduction steps

1.docker run -it --rm --name rabbitmq -p 5672:5672 -p 15672:15672 rabbitmq:3.11-management
2. rabbitmq-plugins enable rabbitmq_stomp
3. netstat -antp | grep 61613 ( Unable to see the post listening)

### Expected behavior

I must see the port 61613 in listening mode

**Comment by michaelklishin** [comment]
Please use GitHub Discussions for questions if the repository has them enabled. They are no longer a new experimental feature on GitHub.

Start with inspecting the logs: do you see any messages related to plugins?
Are you running steps 2 and 3 in the host? If so, it's not going to work unless local CLI tools [can connect and authenticate](https://rabbitmq.com/cli.html#authentication) to the node running in a container. So if you assume that you can just run CLI tools in the host against a node in a container by default, your assumptions are incorrect.

The community Docker image's [own docs](https://hub.docker.com/_/rabbitmq/) suggest that you either run `rabbitmq-plugins` in your own `Dockerfile` built on top, or mount a [preconfigured list of enabled plugins](https://rabbitmq.com/plugins.html#enabled-plugins-file). The latter is the recommended option for almost all automation scenarios.

**Comment by JanEggers** [comment]
-p 61613:61613
